### PR TITLE
update the Profile-Head-of-Resume

### LINKED ISSUE #18: Profile - Head of resume

### PR DESCRIPTION
**Pull request by ajstyle2410**
add the  static image and add name ,email ,contact and address
closes #18

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,39 +9,54 @@
...
+                  <span class="material-symbols-outlined"> person </span>
+                  <p class="text-capitalize">ajay sambhaji deshmukh.</p>
+                </div>
+                <div class="input-group mb-1">
+                  <span class="material-symbols-outlined"> phone_in_talk </span>
+                  <p>8317237646</p>
+                </div>
+                <div class="input-group mb-1">
+                  <span class="material-symbols-outlined"> mail </span>
+                  <p class="text-lowercase">ajdeshmukh241@gmail.com</p>
```
Comment (dipikakoli26) [inline index.html:45]: The email link should be open their default email person.

File: index.html [hunk 2]
```diff
@@ -9,39 +9,54 @@
...
-            </section>
-    </div>
+        <div class="row mt-3">
+          <div class="col-sm-3 mb-3 mb-sm-0">
+            <div class="card border-0">
+              <div class="profile-card text-center">
+                <img
+                  src="./images/profile-image.jpg"
+                  alt="Profile Image"
+                  id="profile-pic"
```
Comment (dipikakoli26) [inline index.html:26]: Its Better to add your own photo at the profile.

**Review by dipikakoli26 — CHANGES_REQUESTED** [review]
@ajstyle2410  Change code as per comments.

**Comment by dipikakoli26** [comment]
@ajstyle2410 Reslove conflict.

**Comment by ajstyle2410** [comment]
conflict is resolved

File: index.html [hunk 3]
```diff
@@ -9,46 +9,60 @@
...
+                  <i class="bi bi-person"></i>
+                  <p class="text-capitalize">ajay sambhaji deshmukh.</p>
+                </div>
+                <div class="input-group mb-1">
+                  <i class="bi bi-telephone"></i>
+                                    <p>8317237646</p>
                 </div>
+                <div class="input-group mb-1">
+                  <i class="bi bi-envelope"></i>
+                  <p class="text-lowercase">ajdeshmukh241@gmail.com</p>
```
Comment (dipikakoli26) [inline index.html:43]: Make this email validate as soon as possible

**Review by dipikakoli26 — CHANGES_REQUESTED** [review]
Change as per suggested comment.

**Pull request merged**